code/script.js

```diff
@@ -257,47 +258,72 @@ const selectQuestion = () => {
 }
 
 // This function should be invoked when you click on 'Find Out' button.
-const checkQuestion = () => {
-  const { category, value } = currentQuestion
-  
+  const checkQuestion = () => {
+  const {category, value} = currentQuestion
 
   // Compare the currentQuestion details with the secret person details in a different manner based on category (hair/eyes or accessories/others).
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
-  let keep = false
 
   if (category === 'hair' || category === 'eyes') {
-    keep = value === secret[category];
+    if (value === secret.hair || value === secret.eyes) {
+      filterCharacters(true)
+    } else {
+      filterCharacters(false)
+    }
+
   } else if (category === 'accessories' || category === 'other') {
-    keep = secret[category].inCludes(value);
+    if ((secret.accessories).includes(value)) {
+      filterCharacters(true)
+    } else if((secret.other).includes(value)) {
```

Comment:
As I mentioned the code above (line 269), we can combine all this code in one line. Instead of listing out all the catagories such as accessories and other. 
`if ((secret.accessories).includes(value))`
and `else if((secret.other).includes(value))`

We can do like this: 
`   if (secret[category].includes(value)) {
      filterCharacters(true);
    } else {
      filterCharacters(false)
    }`
    
    Now the category is either accessories or other